Disable silent rules.

Use Apple libtool instead of ar so that the static library is created
correctly when building universal.
--- src/Makefile.orig	2020-05-30 14:03:15.000000000 -0500
+++ src/Makefile	2021-09-26 05:30:05.000000000 -0500
@@ -145,8 +145,7 @@
 endef # PROG_template
 
 ../wrk/%.o: %.c
-	@echo $<
-	@$(CC) -c $(CFLAGS) -o $@ $<
+	$(CC) -c $(CFLAGS) -o $@ $<
 
 ../bin:
 	@$(call MKDIR,$@)
@@ -154,7 +153,7 @@
 $(eval $(call OBJS_template,common))
 
 ../wrk/common/common.a: $(common_OBJS)
-	$(AR) r $@ $?
+	libtool -static -o $@ $^
 
 $(foreach prog,$(PROGS),$(eval $(call PROG_template,$(prog))))
 
